iml-agent/src/action_plugins/lpurge.rs

```diff
@@ -26,10 +26,9 @@ pub struct Config {
     mailbox: String,
 }
 
-impl fmt::Display for Config {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(
-            f,
+impl Config {
+    fn generate_unit(&self, socket: String) -> String {
```

Comment:
I think this could be unified with `expand_path_fmt`: we just format strings, file name of content.

Reply:
Can you expand on how the unification would work? One item is generating a systemd unit the other is generating a path with optional values.

Are you suggesting to use strfmt for both fns?

Reply:
Yes, strfmt for both. Even maybe in one pass, like `cfg.write(socket, file_template);`